app/models/enums.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+
+import pycountry
+
+
+class Nationality(str, Enum):
+    @classmethod
+    def choices(cls):
+        return [(country.alpha_2, country.name)
```

Comment:
what happens if not found, an exception is thrown but u did not handle that

Reply:
did not even think about that. But it makes since to have it